xdc-plugin/tests/xc7a35tcsg324-1.json

```diff
@@ -0,0 +1,459 @@
+{
```

Comment:
Should we make this a smaller file with only what is required?

Reply:
We could.

Reply:
Create PR #5. However it depends on this PR.